### DISCUSSION TITLE
how to config apifolder  to dynamic folder,since i'm using nexti18-router

### DISCUSSION THREAD

**Discussion by Dmaziyo** [jellydn/next-swagger-doc#982] [Q&A] [answered]
![Snipaste_2024-03-16_15-06-24](https://github.com/jellydn/next-swagger-doc/assets/74136983/4e98eaf5-4262-4f1f-9b40-52d768168863)
![Snipaste_2024-03-16_15-06-33](https://github.com/jellydn/next-swagger-doc/assets/74136983/93112d11-bc9d-4fb9-b93c-1e61f9fdf5e3)
i'm a new here, i don't how to config it,Can anyone help me out?😀

**Comment by jellydn** [comment, marked as answer]
Hi @Dmaziyo Could you try with `apiFolder: app/**/api`? I think it should work.

**Reply by Dmaziyo** [reply]
yeah, it really works.😃
